bionemo-recipes/recipes/codonfm_ptl_te/src/models/encodon_te_pl.py

```diff
@@ -118,6 +117,10 @@ def __init__(  # noqa: D107
         # Best validation loss tracking (simplified for basic loss monitoring)
         self.best_val_loss = float("inf")
 
+        if use_downstream_head and lora_dropout != 0.0:
```

Comment:
do you know what codonfm was using for dropout previously? is this a huge issue to not support it? Just wondering how urgent a PR into peft would be

Reply:
0.1, i saw the loss curve it's the basically the same. but yeah a fix would be nice